schedule/yast/minimal+base/minimal+base@yast-s390x-zkvm.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name:           minimal+base@yast-s390x-zkvm
+description:    >
+  Select a minimal textmode installation by starting with the default and unselecting all patterns
+  except for "base" and "minimal". Not to be confused with the new system role "minimal" introduced with SLE15.
+vars:
+  PATTERNS: base,minimal
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/select_patterns_and_packages
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - boot/reconnect_mgmt_console
+  - console/system_prepare
+  - installation/first_boot
+  - update/zypper_up
```

Comment:
Why we've added some more modules to validate installation, but why only for s390x?
Secondly, what's about reusing `schedule/yast/minimal+base/minimal+base@yast-s390x.yaml` for both? What are the differences why we need another schedule?